config/routes.rb

```diff
@@ -9,6 +9,7 @@
 
   # searches and autocompletes
   post 'match/artworks'
+  get 'match/filter_artworks'
```

Comment:
**suggestion / non-blocking**: Since we're proxying to Gravity's `/api/v1/filter/artworks` endpoint, should we define this as `filter/artworks` or is the `match/` namespace more expected within Rosalind?

Reply:
I debated this as well, then went with the path of least resistance.

But I agree that this would be more intuitive! So I switched this up in 3a0b6c03192446f59110e8c8f893f5f87ba43f2b